File: x/psets/6/sentimental-mario-more/mario.py
# TODO
# import get int
from cs50 import get_int

# get input
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Loop
for i in range(1, height + 1):
    for j in range(height - i):
        print(" ", end="")
    print("#" * (i) + "  " + "#" * (i))

File: x/psets/1/mario-less/mario.c
/*
    JONASH MARCELINO
    CS50
*/
#include <cs50.h>
#include <stdio.h>

// Function declaration
void print_symbol(char, int);

int main(void)
{
    // Get the height of the pyramid
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print the pyramid
    for (int i = 1; i <= height; i++)
    {
        print_symbol(' ', height - i);
        print_symbol('#', i);
        printf("\n");
    }
    return 0;
}

/*
    Function to print a symbol/char.
    c is the char to print.
    i is how many it will print.
*/
void print_symbol(char c, int n)
{
    for (int i = n; i > 0; i--)
    {
        printf("%c", c);
    }
}